rules/config

```diff
@@ -131,3 +131,17 @@ ENABLE_RESTAPI = n
 # ENABLE_NAT - build docker-sonic-nat for nat support
 ENABLE_NAT = y
 
+# INSTALL_KUBERNETES - if set to y kubernetes packages are installed to be able to
+# run as worker node in kubernetes cluster.
+# INSTALL_KUBERNETES = y
```

Comment:
```suggestion
INSTALL_KUBERNETES = n
```

Reply:
What is the difference ?
I see the current behavior as commented, not as set to 'n'.
Can you explain, why we should do this change?

Reply:
You can see some samples above, instead of comment out, we default it to 'n'

---
In reply to: [405760402](https://github.com/Azure/sonic-buildimage/pull/4374#discussion_r405760402) [](ancestors = 405760402)

Reply:
done